bin/mail/2016-08-09-hoc-latam/send-real-emails

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+#3661
```

Comment:
Makes a lot of sense!

On Mon, Aug 8, 2016 at 5:39 PM, Andrew Oberhardt <notifications@github.com>
wrote:

> In bin/mail/2016-08-09-hoc-latam/send-real-emails
> <https://github.com/code-dot-org/code-dot-org/pull/9915#discussion_r73981754>
> :
>
> > @@ -0,0 +1,7 @@
> > +#!/bin/sh
> > +
> > +#3661
>
> Yeah, it's an upper limit. I gave some padding in case we have more
> contacts since my test query (from the 2016 signup) when I actually run the
> scripts.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/9915/files/8a00fcdde19a3539cd5ecc87e35bba7d893fce70#r73981754>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1OEpt8lXY-dGdNMpvUmLgVTFdjGD4Cks5qd8xagaJpZM4JflV7>
> .
>



-- 
Tanya Parker
Product Manager, Code.org
+1 (562) 318-4137